sdk/src/wallet/account/operations/balance.rs

```diff
@@ -19,10 +20,13 @@ use crate::{
     },
 };
 
-impl Account {
+impl<S: 'static + SecretManage> Account<S>
+where
+    Error: From<S::Error>,
+{
     /// Get the balance of the account.
-    pub async fn balance(&self) -> crate::wallet::Result<Balance> {
-        log::debug!("[BALANCE] balance");
+    pub async fn balance(&self) -> Result<Balance> {
+        log::debug!("[BALANCE] get balance");
```

Comment:
I am okay with adding `get` in general, but it's not done consistently atm (see `addresses_balance`). Don't know how much effort it would be to make it consistent across the whole code base. Maybe different PR that fixes some logging?